app/views/email_alert_subscriptions/new.html.erb

```diff
@@ -22,9 +22,9 @@
         items: @signup_presenter.choices_formatted,
         visually_hide_heading: true
       } %>
-      <% if @error_message.present? %>
-        <p class="signup-choices__message"><%= @error_message %></p>
-      <% end %>
+    <% end %>
+    <% if @error_message.present? %>
```

Comment:
Is the intent of this change to catch users submitting weird parameters in the URL?

Reply:
Yes